docs/SECURITY.md

```diff
@@ -0,0 +1,306 @@
+# Security Model of drand
+
+## Table of Contents
+
+* [Notations](#notations)
+* [Model](#model)
+   * [Distributed Key Generation security model](#distributed-key-generation-se
+-model)
+   * [Randomness generation model](#randomness-generation-model)
+* [Attacks](#attacks)
+   * [Randomness Generation](#randomness-generation)
+      * [DDoS the drand network](#ddos-the-drand-network)
+      * [Corruption of the drand network](#corruption-of-the-drand-network)
+   * [Distributed Key Generation](#distributed-key-generation)
+      * [DDoS attacks](#ddos-attacks)
+      * [Corruption attacks](#corruption-attacks)
+      * [Broadcast Channel Assumption](#broadcast-channel-assumption)
+* [TO REVIEW:](#to-review)
+
+## Notations
+
+**Drand node**: a node that is running the drand daemon and participating to the
+creation of the randomness. Each drand node have a longterm public key and a
+private share (after running the setup/resharing phase). The drand network is
+the set of drand nodes connected with each other.
+
+**Relay node**: a node that is connected to a drand daemon and exposing a
+Internet-facing interface allowing to fetch the public randomness. The relay
+network is the set of relay nodes, partially / potentially connected with each
+other.
+
+When the type of the node is not specified in the document, it is assumed from
+the context - most often it refers to a drand node.
+
+**Corrupted node**: a node who is in the control of an attacker. In this case
+the attacker has access to all the cryptographic material this node posses as
+well as the networking authorization. For example, if a relay node is corrupted,
+an attacker has a direct connection to a drand node.
+
+**Offline node**: a node who is unreachable from an external point of view. It
+can be offline from the point of view of another drand node or a relay node. The
+document tries to clarify in which context when relevant.
+
+**Alive node**: a node which is running the binary and sends packets out to the
+inernet.
+
+## Model
+
+In drand, there are two phases which do not require the same security
+assumptions. This section highlights both models and the practical realization
+or assumptions taken.
+
+### Distributed Key Generation security model
```

Comment:
```suggestion
### Distributed Key Generation Ceremony (setup phase)
```

Reply:
So i would like to discuss change of terminology explicitely - I am not sure about the use of "ceremony" for this - 
For trusted setup ceremony, it's ONE TIME THING. Here, DKG happens multiple time over the life of the chain, and it should happen multiple times, even between the same participants to refresh the share. 
Ceremony has this whole context coming from blockchains which I'm not sure we should bear the weight...

Reply:
Ok you convinced me :)  - then we should use that terminology everywhere then.